README.md

```diff
@@ -228,6 +228,9 @@ deploy workloads.
 ### Cluster-wide proxy 
 WMCO does not support adding Windows workloads using a [cluster-wide proxy](https://docs.openshift.com/container-platform/latest/networking/enable-cluster-wide-proxy.html) config for the OpenShift Container Platform. WMCO will not be able to automatically route proxy connections for Windows workloads.
 
+### Accessing secure registries
+WMCO does not support pulling images from secure private registries on the Windows nodes. It is recommended to use images from public registries or pre-pull the images in the VM golden image.
```

Comment:
I'd drop the word golden since this is the general readme, not vsphere specific

Reply:
also nit: line length

Reply:
+1